fix: match more chars for the next.js bug fix (#561)
cloudflare/next-on-pages: .changeset/wicked-shoes-crash.md

---
'@cloudflare/next-on-pages': patch
---

> Update the matcher for the temp fix for a Next.js bug to match any non-whitespace chars.

Author: ```suggestion
Update the matcher for the temp fix for a Next.js bug to match more chars.
```
